cmd/juju/charmhub/data.go

```diff
@@ -51,7 +51,9 @@ type Channel struct {
 	Size       int      `json:"size" yaml:"size"`
 	Version    string   `json:"version" yaml:"version"`
 	Arches     []string `json:"architectures" yaml:"architectures"`
-	Series     []string `json:"series" yaml:"series"`
+	// TODO(juju3) - remove series
+	Series   []string `json:"series" yaml:"series"`
+	Channels []string `json:"channels" yaml:"channels"`
```

Comment:
What is doing to be here? A simple replacement of Channel for Series could be ambiguous depending on the OS.

It's also extremely confusing to have a Channels object in a Channel struct which are completely different. Suggest OSVersion which is a combo of name/channel??  

Saying "7" in the case of centos7 is unhelpful

Reply:
Yeah, this should probably be []Base where it results in "ubuntu/20.04, ubuntu/22.04" etc. I'll fix